src/deadline/unreal_submitter/unreal_logger/handlers.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+
+class UnrealLogHandler(logging.Handler):
```

Comment:
Oh, you did do what I was describing above!  Are there a couple different solutions now though?  Did we need both?

Reply:
What do you mean?

Reply:
We have both UnrealLogHandler handling optional unreal logging in one way and UnrealRenderStepHandler doing using that other approach of if unreal:  unreal.log - I was expecting everything to use the same approach

Reply:
Ok, in that way we could try to add logger creation in the top of render_step_handler depending on unreal import result

Reply:
That we be great if we could be more consistent without too much trouble!